.gitignore

```diff
@@ -3,6 +3,7 @@
 **/target
 *.pyc
 **/*.rs.bk
+yarn.lock
```

Comment:
We sure about this? I don't know js enough but it wasn't done so far, any reason why? If we really ignore it, we should also remove the ones that have been pushed, apparently 4

Reply:
On project level, it's fine. Inside docs, while testing one extra one got created. So, removed it.

Reply:
I would only add an ignore in the example folder. All other lock files should still be committed, right?

Reply:
Looks like this is back?